pwa/.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    semi: false,
+    trailingComma: 'all',
+    singleQuote: true,
+    printWidth: 80,
```

Comment:
@abpwrs this has been changed to 80. 120 was a mistake, I have my editor ruler at 80 and didn't notice.

This resolves your previous comment, if you could add your approval I'd appreciate it.